### ISSUE TITLE
groupsocketlisten with conn_sep branch of knxd

### ISSUE THREAD

**Issue by snowdd1** [andreek/node-eibd#26]
Hi André, 
have you tried the groupsocketlisten with the latest knxd? I cannot get any telegrams from the bus. Writing works as before, but no answers.
I had my house upgraded recently and all works, except groupsocketlisten from node-eibd.

BTW: I found the actual groupsocketlisten has a flaw - since the introduction of more `err` objects the callbacks that do not use an err object might get mixed. openGroupSocket(readOnly, callback) expects the callback have a signature of function(parser), the callback used has a signature of function(err, parser). - I'll put an PR for that, but that doesn't seam to solve the issue.

Regards
Raoul

**Comment by andreek** [comment]
Hi Raoul,

unfortunately I didn't get my hands on the latest knxd yet. I'm kind of busy the last weeks/months. 

Can you read messages from the bus with other knx/eibd tools? Maybe native groupsocketlisten?

Regards,
Andree

**Comment by snowdd1** [comment]
Hi Andree,
after writeing this issue I tried with the native knxd tools (knxtool) - vbusmonitor1 works, groupsocketlisten doesn't answer anything. I opened an issue with knxd. 
For the time being I need to return to the more stable master branch of knxd, I hope that is working better at that. As an intermediate attempt I will set up a second knxd on my node raspberry from the master branch, maybe that can talk to the newer version and provide a groupsocketlisten. Cannons and sparrows, I know.
Raoul
